src/main/java/io/netty/incubator/codec/http3/Http3FrameCodec.java

```diff
@@ -319,44 +357,81 @@ private Http3SettingsFrame decodeSettings(ChannelHandlerContext ctx, ByteBuf in,
      * <p>
      * This method assumes the entire header block is contained in {@code in}.
      */
-    private boolean decodeHeaders(ChannelHandlerContext ctx, Http3Headers headers, ByteBuf in, boolean trailer) {
+    private boolean decodeHeaders(ChannelHandlerContext ctx, Http3Headers headers, ByteBuf in, int length,
```

Comment:
nit: I know it was not there before (lazy me...) but consider adding some javadocs to explain the meaning of the return value